src/vita-elf.c

```diff
@@ -31,20 +31,27 @@
 
 static void free_rela_table(vita_elf_rela_table_t *rtable);
 
-static int load_stubs(Elf_Scn *scn, int *num_stubs, vita_elf_stub_t **stubs)
+static int load_stubs(Elf_Scn *scn, int *num_stubs, vita_elf_stub_t **stubs, char *name)
 {
 	GElf_Shdr shdr;
 	Elf_Data *data;
 	uint32_t *stub_data;
 	int chunk_offset, total_bytes;
 	vita_elf_stub_t *curstub;
+	int old_num;
 
 	gelf_getshdr(scn, &shdr);
 
-	*num_stubs = shdr.sh_size / 16;
-	*stubs = calloc(*num_stubs, sizeof(vita_elf_stub_t));
-
+	old_num = *num_stubs;
+	*num_stubs = old_num + shdr.sh_size / 16;
+	*stubs = realloc(*stubs, *num_stubs * sizeof(vita_elf_stub_t));
```

Comment:
Make sure my single realloc is equal to the previous code which had a special case when *num_stubs == 0.